_data/schedule.yml

```diff
@@ -1,164 +1,154 @@
 - 
-  date: "Aug 27"
+  date: "Jan 14"
   materials: "LaValle, Ch 1"
-  title: "[Course overview, defining VR, historical perspective](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture1-Introduction.pptx)"
+  title: "[Defining VR, course overview, course project expectations](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture1-Introduction.pptx)"
 
 - 
-  date: "Aug 29"
+  date: "Jan 16"
   materials: "LaValle, Ch 2.1, 2.2"
-  title: "[Bird's-eye view: VR hardware](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture2.ppt)"
+  title: "[Design Processes - Basic Principles and Top Down Design Processes](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture2.ppt)"
 
 - 
-  date: "Sep 3"
+  date: "Jan 21"
   materials: ~
-  title: "No lecture"
+  title: "**No Lecture: M. L. King Day**"
 
 - 
-  date: "Sep 5"
+  date: "Jan 23"
   materials: "LaValle, Ch 2.3, 3"
-  title: "[Bird's-eye view: Sensation and perception](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/lecture3.pptx)"
+  title: "[3D Computer Graphics: Geometric Modeling](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/lecture3.pptx)"
 
 - 
-  date: "Sep 10"
+  date: "Jan 28"
   materials: "LaValle, Ch 3.2, 3.3"
-  title: "[Geometric modeling, linear transformations, 2D rigid-body transformations](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture4.pptx)"
+  title: "[Design Processes – VR Design (General Considerations)](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture4.pptx)"
```

Comment:
what about "Design Processes - General VR considerations" or similar?

Reply:
Yeah - I'll change to colons instead of parentheses